app/views/main/_boolean.html.haml

```diff
@@ -1 +1,11 @@
 = form.input question['name'], as: :radio_buttons, collection: [['Yes', 1], ['No', 0]], label: question['title']
+
+:javascript
```

Comment:
This seems a little bit specific to be put into a shared template. I'll see if we can be a bit more generic